drv/stm32h7-spi-server/src/main.rs

```diff
@@ -81,18 +81,19 @@ fn main() -> ! {
     // Configure all devices' CS pins to be deasserted (set).
     // We leave them in GPIO output mode from this point forward.
     for device in CONFIG.devices {
-        sys.gpio_set_reset(device.cs.port, device.cs.pin_mask, 0)
+        for pin in device.cs {
+            sys.gpio_set_reset(pin.port, pin.pin_mask, 0).unwrap();
```

Comment:
Use `sys.gpio_set(pin)` and `sys.gpio_configure_output(pin, ...)` here instead of the fully general functions